dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapperTest.java

```diff
@@ -303,6 +305,7 @@ void shouldMapAttributeFilters() throws BadRequestException, ForbiddenException
                     List.of(new QueryFilter(QueryOperator.EQ, "2")),
                     TEA_2_UID,
                     List.of(new QueryFilter(QueryOperator.LIKE, "foo"))))
+            .orgUnitMode(SELECTED)
```

Comment:
How can it be selected when there is no orgUnit?

Reply:
nice one, this one should be accessible